app/src/main/java/com/randomappsinc/studentpicker/home/MainActivity.java

```diff
@@ -309,4 +329,18 @@ public boolean onOptionsItemSelected(MenuItem item) {
         }
         return super.onOptionsItemSelected(item);
     }
+
+    @Override
+    public void onRenameListConfirmed(int position, String newListName) {
```

Comment:
nit: Move this under onItemEditClick to keep similar code next to each other and improve readability.